frontend/src/lib/components/CommandBar/SearchResults.tsx

```diff
@@ -10,36 +10,39 @@ export const SearchResults = (): JSX.Element => {
         useValues(searchBarLogic)
 
     return (
-        <div className="flex grow overflow-hidden">
-            <div className="w-1/2 overscroll-none overflow-y-auto border-r bg-bg-light">
-                {combinedSearchLoading && (
-                    <>
-                        <SearchResultSkeleton />
-                        <SearchResultSkeleton />
-                        <SearchResultSkeleton />
-                    </>
-                )}
-                {!combinedSearchLoading && combinedSearchResults?.length === 0 && (
-                    <div className="h-full flex flex-col items-center justify-center p-3">
-                        <h3 className="mb-0 text-xl">No results</h3>
-                        <p className="opacity-75 mb-0">This doesn't happen often, but we're stumped!</p>
-                        <DetectiveHog height={150} width={150} />
+        <div className="grow" style={{ height: 'calc(100% - 2.375rem)' }}>
```

Comment:
Could this be better as a CSS class? Why the 2.375rem

Reply:
Converted it to a class. This is to offset the input, so that 100% height, is actually 100% of the remaining space. That's important for the scrolling, as otherwise other ancestors (= left bar) scroll as well and the input gets hidden.